src/common/dsp/utilities/DSPUtils.h

```diff
@@ -365,12 +367,15 @@ inline double hamming(int i, int n)
     return 0.54 - 0.46 * cos(2 * M_PI * i / (n - 1));
 }
 
-inline char *float_to_str(float value, char *str)
+inline std::string float_to_clocalestr(float value)
```

Comment:
I think changing this name is definitely a good idea, but I'd add a comment too saying why we've got this function. It already tripped someone up (you!) when it came to updating it to use std::string!

Reply:
Yes, valid suggestion, makes total sense. Will add comment 🙂